### ISSUE TITLE
[request] save settings globally

### ISSUE THREAD

**Issue by Rihardskombu** [polywock/globalSpeed#324]
hi, **the extension itself is just perfect.**
it is sad that I have to configure my settings over, over and over again when switching to new device / re-installing my OS.

there is a different extension (**_streamkeys_**), which I configured only once, and all my settings are synced by default across different devices.

**Comment by Rihardskombu** [comment]
my request is duplicate of https://github.com/polywock/globalSpeed/issues/188
